diff --git a/dyninstAPI/src/pcEventHandler.C b/dyninstAPI/src/pcEventHandler.C
index 46a4ff9..64c516d 100644
--- a/dyninstAPI/src/pcEventHandler.C
+++ b/dyninstAPI/src/pcEventHandler.C
@@ -839,6 +839,8 @@ bool PCEventHandler::handleLibrary(EventLibrary::const_ptr ev, PCProcess *evProc
     const set<Library::ptr> &added = ev->libsAdded();
 
     for(set<Library::ptr>::const_iterator i = added.begin(); i != added.end(); ++i) {
+        if ((*i)->getAbsoluteName() == "linux-vdso.so.1")
+            continue;
         Address dataAddress = (*i)->getLoadAddress();
         if( evProc->usesDataLoadAddress() ) dataAddress = (*i)->getDataLoadAddress();
 
